src/main/resources/db/migration/V1_24__operator.sql

```diff
@@ -0,0 +1,194 @@
+CREATE TABLE operator
+(
+    id          SERIAL PRIMARY KEY,
+    name VARCHAR(80) NOT NULL
+);
+
+INSERT INTO operator(name)
+VALUES ('PSP'),
+       ('G4S'),
+       ('Serco'),
+       ('Sodexo'),
+       ('MTCNovo');
+
+CREATE TABLE prison_operator
+(
+    prison_id VARCHAR(6),
```

Comment:
Think we need foreign keys on prison_id and operator_id

Reply:
Agree, will update